cmdutil/service/standard.go

```diff
@@ -79,6 +79,14 @@ func New(appConfig interface{}, ofs ...OptionFunc) *Standard {
 	s.Add(debug.New(logger, sc.Debug.Port))
 	s.Add(signals.NewServer(logger, syscall.SIGINT, syscall.SIGTERM))
 
+	if o.enableOpenCensusTracing {
```

Comment:
Could this be more automatic based on presence of config, similar to how rollbar integration in the logger is skipped if config is missed? Maybe explicit enabling is fine to start but would be nice to keep a more on-by-default setup in mind.

Reply:
On by default is a goal, but IMO it's too soon for that. Technically enabling it by default is also a breaking API change (although that probably doesn't matter atm either).